common/src/main/java/net/william278/husksync/database/MongoDbDatabase.java

```diff
@@ -64,14 +65,22 @@ public MongoDbDatabase(@NotNull HuskSync plugin) {
     public void initialize() throws IllegalStateException {
         final Settings.DatabaseSettings.DatabaseCredentials credentials = plugin.getSettings().getDatabase().getCredentials();
         try {
-            mongoConnectionHandler = new MongoConnectionHandler(
-                    credentials.getHost(),
-                    credentials.getPort(),
-                    credentials.getUsername(),
-                    credentials.getPassword(),
-                    credentials.getDatabase(),
-                    credentials.getMongoAuthDb()
-            );
+            if (plugin.getSettings().getDatabase().getType() == Type.MONGO_ATLAS) {
+                String baseURI = "mongodb+srv://{0}:{1}@{2}/?retryWrites=true&w=majority";
+                baseURI = baseURI.replace("{0}", credentials.getUsername());
+                baseURI = baseURI.replace("{1}", credentials.getPassword());
+                baseURI = baseURI.replace("{2}", credentials.getHost());
```

Comment:
```suggestion
            final Database.Type type = plugin.getSettings().getDatabase().getType();
            if (type == Type.MONGO_ATLAS) {
                final String uri = String.format("%s://%s:%s@%s/?retryWrites=true&w=majority",
                type.getProtocol(), credentials.getUsername(), credentials.getPassword(), credentials.getHost());

Reply:
I was going to switch to String.format, but my way makes more sense in the new implementation